cli/src/main/java/io/kestra/cli/AbstractCommand.java

```diff
@@ -207,14 +207,13 @@ private void startWebserver() {
                     try {
                         endpoint = UriBuilder.of(server.getURL().toURI())
                             .port(this.endpointConfiguration.getPort().get())
-                            .path("/health")
```

Comment:
This was specifically asked by @tchiotludo so he can click on the log from the console and it opens the health endpoint so I think he may be sad if you remove it (but I agree, it's strange).